pkgs/by-name/my/myks/package.nix

```
@@ -0,0 +1,49 @@
+{ lib, buildGoModule, fetchFromGitHub, testers, installShellFiles, myks, stdenv
+}:
```

One item per line will make the diffs smaller in the future.

```suggestion
{ lib
, stdenv
, buildGoModule
, fetchFromGitHub
, testers
, installShellFiles
, myks
}:
```